plugins/plugin-js/src/main/kotlin/com/kotlin/aws/js/runtime/tasks/BuildLambda.kt

```diff
@@ -0,0 +1,34 @@
+package com.kotlin.aws.js.runtime.tasks
+
+import com.kotlin.aws.js.runtime.utils.runtime
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+open class BuildLambda : DefaultTask() {
+
+    @get:Input
+    val extension = project.runtime
+
+    init {
+        group = "aws"
+    }
+
+    @TaskAction
+    fun generate() {
+        val dir = if (extension.outputDir != null)
```

Comment:
extract extension.outputDir as an variable
val output = extension.outputDir. after that you dont need the `!!`